app/Domain/Clients/Models/Clients.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Leantime\Domain\Clients\Models {
+
+    class Clients
+    {
+        public $id;
+        
+        public $name;
+        
+        public $street;
+        
+        public $zip;
+        
+        public $city;
+        
+        public $state;
+        
+        public $country;
+        
+        public $phone;
+        
+        public $internet;
+        
+        public $email;
+        
+        public $numberOfProjects;
+        
+        public function __construct() {}
```

Comment:
You could refactor the constructor to take an array and then map the array to your client model. See the ticket model for example